packages/common/src/core/schemas/internal/getFeaturedImageUrl.ts

```diff
@@ -0,0 +1,14 @@
+import { IArcGISContext } from "../../../ArcGISContext";
+import { cacheBustUrl } from "../../../urls/cacheBustUrl";
+import { HubEntity } from "../../types/HubEntity";
+
+export function getFeaturedImageUrl(
+  entity: HubEntity,
+  context: IArcGISContext
+) {
+  const queryParams = context.isAuthenticated
+    ? `?token=${context.session.token}`
+    : "";
+  // TODO: Decide if the url should be passed in or plucked out of this deep path here
```

Comment:
I think this is a good question, and generally speaking, I feel like it would be nice to make this util more generic/less featured-image-specific. I just ran into needing similar functionality with Aaron who's trying to hook up editing an item's thumbnail. The differences there are: 
1. the url lives at `entity.thumbnailUrl`
2. `entity.thumbnailUrl` already has the token appended to it

So not entirely sure how to make this generic to work in both cases, but you could do something like: 

```ts
getImageUrl (
  url: string,
  context?: IArcGISContext // make this optional if you want to append the token
) {
  const queryParams = context?.isAuthenticated
      ? `?token=${context.session.token}`
      : "";

  return cacheBustUrl(`${url}${queryParams}`);
}

Reply:
I agree - I'll make an issue to try to resolve this - I don't want to add additional concerns / refactors to this PR